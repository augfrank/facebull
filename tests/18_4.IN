M0   C0   C1  2928
M1   C0   C2  2699
M2   C0   C4  2621
M3   C0   C8  2490
M4   C0   C9  2831
M5   C0   C12  2476
M6   C0   C13  2130
M7   C0   C14  2598
M8   C0   C15  2022
M9   C0   C16  2310
M10   C0   C17  2843
M11   C1   C0  2426
M12   C1   C3  2645
M13   C1   C6  2470
M14   C1   C9  2027
M15   C1   C10  2961
M16   C1   C12  2476
M17   C1   C14  2921
M18   C1   C16  2538
M19   C1   C17  2585
M20   C2   C0  2944
M21   C2   C6  2335
M22   C2   C7  2907
M23   C2   C8  2264
M24   C2   C9  2480
M25   C2   C12  2819
M26   C3   C1  2725
M27   C3   C2  2230
M28   C3   C4  2668
M29   C3   C5  2695
M30   C3   C6  2725
M31   C3   C7  2688
M32   C3   C8  2534
M33   C3   C9  2605
M34   C3   C10  2789
M35   C3   C11  2673
M36   C3   C13  2331
M37   C3   C14  2754
M38   C3   C15  2883
M39   C3   C16  2613
M40   C3   C17  2696
M41   C4   C2  2951
M42   C4   C7  2779
M43   C4   C8  2142
M44   C4   C9  2568
M45   C4   C10  2362
M46   C4   C11  2836
M47   C4   C12  2402
M48   C4   C16  2597
M49   C4   C17  2827
M50   C5   C0  2603
M51   C5   C3  2146
M52   C5   C7  2757
M53   C5   C9  2505
M54   C5   C10  2125
M55   C5   C11  2327
M56   C5   C13  2167
M57   C5   C17  2729
M58   C6   C0  2642
M59   C6   C1  2647
M60   C6   C3  2618
M61   C6   C7  2960
M62   C6   C10  2913
M63   C6   C12  2039
M64   C6   C13  2341
M65   C6   C15  2166
M66   C7   C0  2910
M67   C7   C2  2030
M68   C7   C5  2013
M69   C7   C9  2100
M70   C7   C11  2817
M71   C7   C15  2118
M72   C7   C16  2971
M73   C7   C17  2399
M74   C8   C0  2496
M75   C8   C1  2358
M76   C8   C4  2580
M77   C8   C9  2787
M78   C8   C10  2411
M79   C8   C11  2029
M80   C9   C4  2236
M81   C9   C7  2730
M82   C9   C12  2144
M83   C9   C13  2184
M84   C9   C15  2333
M85   C9   C17  2066
M86   C10   C0  2566
M87   C10   C4  2054
M88   C10   C5  2962
M89   C10   C7  2903
M90   C10   C8  2647
M91   C10   C11  2226
M92   C10   C12  2674
M93   C10   C15  2408
M94   C10   C16  2985
M95   C11   C0  2118
M96   C11   C1  2564
M97   C11   C4  2882
M98   C11   C5  2252
M99   C11   C6  2643
M100   C11   C9  2191
M101   C11   C14  2168
M102   C12   C0  2966
M103   C12   C3  2786
M104   C12   C5  2252
M105   C12   C6  2635
M106   C12   C10  2194
M107   C12   C11  2598
M108   C12   C16  2484
M109   C12   C17  2826
M110   C13   C4  2951
M111   C13   C9  2145
M112   C13   C10  2739
M113   C13   C14  2688
M114   C13   C15  2946
M115   C13   C17  2984
M116   C14   C0  2003
M117   C14   C1  2152
M118   C14   C5  2923
M119   C14   C7  2301
M120   C14   C8  2646
M121   C14   C9  2326
M122   C14   C10  2230
M123   C14   C11  2861
M124   C14   C12  2386
M125   C14   C13  2258
M126   C14   C17  2046
M127   C15   C1  2312
M128   C15   C3  2100
M129   C15   C5  2272
M130   C15   C12  2176
M131   C15   C13  2489
M132   C15   C17  2249
M133   C16   C2  2214
M134   C16   C6  2027
M135   C16   C10  2226
M136   C16   C11  2404
M137   C16   C13  2684
M138   C16   C15  2145
M139   C17   C0  2253
M140   C17   C2  2575
M141   C17   C3  2230
M142   C17   C4  2777
M143   C17   C11  2459
M144   C17   C13  2616
M145   C17   C14  2818
