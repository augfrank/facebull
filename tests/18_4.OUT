39240
8 18 23 27 42 56 63 68 79 85 87 101 106 111 117 128 134 139
